.github/workflows/linting.yml

```diff
@@ -5,13 +5,18 @@ on:
     branches: [ main, develop ]
   pull_request:
     branches: [ main, develop ]
+    paths:
+      - '.github/workflows/linting.yml'
+      - '**.cpp'
+      - '**.hpp'
+      - '**CMakeLists.txt'
 
 jobs:
   clang-format:
     runs-on: ubuntu-24.04
     steps:
     - uses: actions/checkout@v2
-    - name: clang-format
+    - name: cmake-format
```

Comment:
Doesn't this change belong in line 29/34?

Reply:
Oops! Good spot - fixed in 470d3bd9.